src/app/profile/tenant/tenant.component.spec.ts

```diff
@@ -52,9 +54,16 @@ describe('TenantComponent', () => {
         { provide: TenantService, useValue: mockTenantService },
         { provide: UserService, useValue: mockUserService },
         { provide: Logger, useValue: mockLogger },
+        { provide: ProviderService, useValue: mockProviderService },
+        { provide: GitHubService, useValue: mockGitHubService },
         { provide: WIT_API_URL, useValue: 'http://example.com'}
       ]
     });
+    TestBed.overrideProvider(AuthenticationService, { useValue: mockAuthenticationService });
+    TestBed.overrideProvider(GettingStartedService, { useValue: mockGettingStartedService });
+    TestBed.overrideProvider(TenantService, { useValue: mockTenantService });
+    TestBed.overrideProvider(ProviderService, { useValue: mockProviderService });
+    TestBed.overrideProvider(GitHubService, { useValue: mockGitHubService });
```

Comment:
Why is this override here and also configured in the above call to `configureTestingModule`?